sdk/search/azure-search-documents/src/samples/java/com/azure/search/documents/indexes/SearchIndexerAsyncClientJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
One thing I have noticed is that our `src/samples` directory containing a lot of `*JavaDocCodeSnippets` files making it very hard for the user to find real samples in the directory. Should we consolidate all javadoc codesnippets into a single Java file? Since this is mostly used by us and putting them all together might not be too bad. 

This allows all the real sample java files to be easily discoverable for the user with just 1 java file for codesnippets that they can ignore.

Another option is to have a separate directory `src/codesnippets`.

Thoughts?

Reply:
Proceed with put all in one file.